diff -Nur a/base-files/files/bin/config_generate b/base-files/files/bin/config_generate
--- a/base-files/files/bin/config_generate	2023-05-29 23:12:17.752212800 +0800
+++ b/base-files/files/bin/config_generate	2023-05-29 23:38:05.013267700 +0800
@@ -321,10 +321,8 @@
 		set system.ntp='timeserver'
 		set system.ntp.enabled='1'
 		set system.ntp.enable_server='0'
-		add_list system.ntp.server='0.openwrt.pool.ntp.org'
-		add_list system.ntp.server='1.openwrt.pool.ntp.org'
-		add_list system.ntp.server='2.openwrt.pool.ntp.org'
-		add_list system.ntp.server='3.openwrt.pool.ntp.org'
+		add_list system.ntp.server='ntp1.aliyun.com'
+		add_list system.ntp.server='time2.cloud.tencent.com'
 	EOF
 
 	if json_is_a system object; then
diff -Nur a/default-settings/files/zzz-default-settings b/default-settings/files/zzz-default-settings
--- a/default-settings/files/zzz-default-settings	2023-05-29 23:12:30.760370100 +0800
+++ b/default-settings/files/zzz-default-settings	2023-05-29 23:54:48.911247700 +0800
@@ -7,8 +7,8 @@
 #uci set luci.main.mediaurlbase=/luci-static/bootstrap_mod
 uci commit luci
 
-uci set system.@system[0].timezone=CST-8
-uci set system.@system[0].zonename=Asia/Shanghai
+uci set system.@system[0].timezone=HKT-8
+uci set system.@system[0].zonename=Asia/Hong Kong
 uci commit system
 
 uci set fstab.@global[0].anon_mount=1
@@ -37,10 +37,6 @@
 uci set network.lan.ip6assign='64'
 uci commit network
 
-sed -i '/option disabled/d' /etc/config/wireless
-sed -i '/set wireless.radio${devidx}.disabled/d' /lib/wifi/mac80211.sh
-wifi up
-
 sed -i '/DISTRIB_DESCRIPTION/d' /etc/openwrt_release
 echo "DISTRIB_DESCRIPTION='OpenWrt SNAPSHOT'" >> /etc/openwrt_release
 
@@ -58,7 +54,6 @@
 /etc/init.d/softethervpnbridge disable 2>/dev/null
 /etc/init.d/softethervpnserver disable 2>/dev/null
 /etc/init.d/softethervpnclient disable 2>/dev/null
-/etc/init.d/haproxy disable 2>/dev/null
 /etc/init.d/kcptun disable 2>/dev/null
 
 chmod 0755 /etc/init.d/*
